pandas/tests/computation/test_eval.py

```diff
@@ -1890,6 +1890,16 @@ def test_negate_lt_eq_le(engine, parser):
         tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("column", ["Timestamp", "datetime"])
```

Comment:
Could you test all global defaults?

Reply:
Hi, 
1) All global defaults list is ['Timestamp','datetime','True','False','list','tuple','inf','Inf','__name__','__doc__',
 '__package__', '__loader__', '__spec__', '__file__', '__cached__',  '__builtins__', 'tm',
 'pandas', 'np', 'DataFrame','column', 'df', 'expected'], there are total 23 variables. 

2) ['True', 'False', 'inf', 'Inf'] variables fail the tests. 

3) Example, if column='False', it will give error `KeyError: 'False: boolean label can not be used without a boolean index'`

What suggestion you recommend to deal with this?

Reply:
I just meant the ones in `DEFAULT_GLOBALS`

Hmm I would just xfail the values that fail for now and raise a separate issue about those